src/internal/formdata-event-polyfill.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```

Comment:
I'm OK with this, but this isn't a copy + paste of the polyfill — I refactored it to be more TypeScript friendly, although I didn't spend a ton of time on it. Hopefully we can remove it soon.

Reply:
Ah good to know. It looked like a copy which is why I added this. Anything you want me to do here?

Reply:
Nah, if I need to touch it again I'll see if removing it requires a lot of refactoring. No sense in wasting time on something that was built to be thrown out.